### PR TITLE
Follow standards

### PR DESCRIPTION
**Pull request by pacospace**
## This introduces a breaking change

- [x] Yes
- [ ] No

## This Pull Request implements

Adjust naming to follow standard across repositories.
Update README file

### CODE REVIEW

File: openshift/secret-template.yaml [hunk 1]
```diff
@@ -39,6 +39,6 @@ objects:
       labels:
         app: srcopsmetrics
     stringData:
-      githubAccessToken: ${GITHUB_ACCESS_TOKEN}
-      thothCephKeyID: ${THOTH_CEPH_KEY_ID}
-      thothCephSecretKey: ${THOTH_CEPH_SECRET_KEY}
+      github-access-token: ${GITHUB_ACCESS_TOKEN}
+      ceph-key-id: ${CEPH_KEY_ID}
+      ceph-secret-key: ${CEPH_SECRET_KEY}
```
Comment (xtuchyna) [inline openshift/secret-template.yaml:44]: so we are using kebab case for the envs?
is somewhere a doc that would describe our standards on openshift? - so i know for the future

File: openshift/configMap-template.yaml [hunk 2]
```diff
@@ -19,19 +19,19 @@ parameters:
   - description: Ceph Bucket name where knowledge is stored.
     displayName: Ceph Bucket
     required: true
-    name: BUCKET
+    name: CEPH_BUCKET
     value: DH-DEV-DATA
 
   - description: Ceph Host name where knowledge is stored.
     displayName: Ceph Host (endpoint)
```
Comment (xtuchyna) [inline openshift/configMap-template.yaml:26]: probably change description to endpoint too?

File: README.md [hunk 3]
```diff
@@ -14,18 +14,46 @@ Pre-Usage
...
+- `CEPH_BUCKET` Ceph Bucket name where knowledge is stored.
+- `CEPH_BUCKET_PREFIX` Ceph Prefix where knowledge is stored.
+- `CEPH_KEY_ID` Ceph Key ID
+- `CEPH_SECRET_KEY` Ceph Secret Key
+
+If you want to test locally you have also the option to store locally without providing any parameter adding `-l` flag:
+
+.. code-block:: console
+
+    GITHUB_ACCESS_TOKEN=<github_acess_token> PYTHONPATH=. pipenv run srcopsmetrics/cli.py --project <project_name> -c -l
```
Comment (xtuchyna) [inline README.md:45]: word 'project' remained here

File: README.md [hunk 4]
```diff
@@ -14,18 +14,46 @@ Pre-Usage
...
 
 Usage - Visualize Project Statistics
 ====================================
 
 .. code-block:: console
 
-    PYTHONPATH=. pipenv run srcopsmetrics/cli.py --project <project_name> -v True
+    PYTHONPATH=. pipenv run srcopsmetrics/cli.py --project <project_name> -v
```
Comment (xtuchyna) [inline README.md:52]: 'project' also here

File: README.md [hunk 5]
```diff
@@ -14,18 +14,46 @@ Pre-Usage
...
+
+.. code-block:: console
+
+    PYTHONPATH=. pipenv run srcopsmetrics/cli.py --organization <project_name> -v
```
Comment (xtuchyna) [inline README.md:56]: <org_name>

File: openshift/secret-template.yaml (hunk 1 above)
Comment (pacospace) [reply openshift/secret-template.yaml:44]: We can maybe write one :) @fridex WDTY? I just follow our ConfigMap like Thoth one.

File: README.md [hunk 6]
```diff
@@ -14,18 +14,46 @@ Pre-Usage
...
+- `CEPH_BUCKET` Ceph Bucket name where knowledge is stored.
+- `CEPH_BUCKET_PREFIX` Ceph Prefix where knowledge is stored.
+- `CEPH_KEY_ID` Ceph Key ID
+- `CEPH_SECRET_KEY` Ceph Secret Key
+
+If you want to test locally you have also the option to store locally without providing any parameter adding `-l` flag:
+
+.. code-block:: console
+
+    GITHUB_ACCESS_TOKEN=<github_acess_token> PYTHONPATH=. pipenv run srcopsmetrics/cli.py --repository <org_name> -c -l
```
Comment (xtuchyna) [inline README.md:45]: just one here <project_name>

File: README.md [hunk 7]
```diff
@@ -14,18 +14,46 @@ Pre-Usage
...
 .. code-block:: console
 
-    GITHUB_ACCESS_TOKEN=<github_acess_token> PYTHONPATH=. pipenv run srcopsmetrics/cli.py --project <project_name> -c True
+    GITHUB_ACCESS_TOKEN=<github_acess_token> PYTHONPATH=. pipenv run srcopsmetrics/cli.py --project <project_name> -c
+
+2. You can extract knowledge from a organization using the following command:
+
+.. code-block:: console
+
+    GITHUB_ACCESS_TOKEN=<github_acess_token> PYTHONPATH=. pipenv run srcopsmetrics/cli.py --organization <project_name> -c
```
Comment (xtuchyna) [inline README.md:27]: <org_name>

File: README.md [hunk 8]
```diff
@@ -14,18 +14,46 @@ Pre-Usage
 Usage - Create Bot Knowledge
 ============================
 
-1. You can extract knowledge from a project using the following command:
+1. You can extract knowledge from a repository using the following command:
 
 .. code-block:: console
 
-    GITHUB_ACCESS_TOKEN=<github_acess_token> PYTHONPATH=. pipenv run srcopsmetrics/cli.py --project <project_name> -c True
+    GITHUB_ACCESS_TOKEN=<github_acess_token> PYTHONPATH=. pipenv run srcopsmetrics/cli.py --project <project_name> -c
```
Comment (xtuchyna) [inline README.md:21]: repository

File: openshift/secret-template.yaml (hunk 1 above)
Comment (fridex) [reply openshift/secret-template.yaml:44]: Yeah, we don't have any guide on this. We can do that, I just follow the rest of the code most of the time :)

**Pull request merged**